packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketLayoutNavigation.tsx

```diff
@@ -1,25 +1,59 @@
-import styled from 'styled-components';
-import { goto } from 'src/actions/suite/routerActions';
-import { useDevice, useDispatch, useSelector } from 'src/hooks/suite';
-import { WalletLayoutNavigation, WalletLayoutNavLink } from 'src/components/wallet';
+import styled, { css } from 'styled-components';
+import { useDevice, useSelector } from 'src/hooks/suite';
 import { getTitleForNetwork } from '@suite-common/wallet-utils';
 import { Translation } from 'src/components/suite';
 import { selectSelectedAccount } from 'src/reducers/wallet/selectedAccountReducer';
 import { FirmwareType } from '@trezor/connect';
+import { FONT_SIZE, FONT_WEIGHT } from '@trezor/components/src/config/variables';
+import { TranslationKey } from '@suite-common/intl-types';
+import { Route } from '@suite-common/suite-types';
+import { borders, spacingsPx } from '@trezor/theme';
+import { NavigationItem } from '../../../../../components/suite/Preloader/SuiteLayout/Sidebar/NavigationItem';
+import { IconName } from '@suite-common/icons';
+import { variables } from '@trezor/components';
 
-const SavingsWalletLayoutNavLinkWrapper = styled.div`
+const List = styled.div`
     display: flex;
-    margin-left: auto;
-    padding-left: 42px;
+    flex-direction: column;
+    width: 171px;
+    gap: ${spacingsPx.xxs};
+
+    ${variables.SCREEN_QUERY.BELOW_LAPTOP} {
+        width: 100%;
+        flex-direction: row;
+        overflow: scroll;
+
+        ::-webkit-scrollbar {
+            display: none;
+        }
+    }
 `;
 
-const items = [
-    { route: 'wallet-coinmarket-buy', title: 'TR_NAV_BUY' },
-    { route: 'wallet-coinmarket-sell', title: 'TR_NAV_SELL' },
-    { route: 'wallet-coinmarket-exchange', title: 'TR_NAV_EXCHANGE' },
-    { route: 'wallet-coinmarket-spend', title: 'TR_NAV_SPEND' },
-] as const;
-const p2pRoute = 'wallet-coinmarket-p2p';
+const NavListItem = styled(NavigationItem)`
+    white-space: nowrap;
+    font-size: ${FONT_SIZE.NORMAL};
+    font-weight: ${FONT_WEIGHT.MEDIUM};
+
+    ${({ isActive }) =>
+        isActive
+            ? css`
+                  border-radius: ${borders.radii.full};
+              `
+            : ''}
+`;
+
+const Divider = styled.div`
+    margin: ${spacingsPx.xxs} 0; /* gap xxs + margin xxs = spacing xs around divider */
+    width: 100%;
+    border-bottom: 1px solid ${({ theme }) => theme.textDefault};
```

Comment:
```jsx
border-bottom: 1px solid ${({ theme }) => theme.borderOnElevation0};
```

Reply:
shall we do map for elevation and borders as well?

Reply:
> shall we do map for elevation and borders as well?

Not today, but after merging to develop yes